docs/bigquery/syncquery/syncquery.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Command syncquery queries a Google BigQuery dataset.
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"cloud.google.com/go/bigquery"
+
+	"golang.org/x/net/context"
+)
+
+// Query returns a slice of the reults of a query.
+func Query(proj, q string) ([]bigquery.ValueList, error) {
+	ctx := context.Background()
+
+	client, err := bigquery.NewClient(ctx, proj)
+	if err != nil {
+		return nil, err
+	}
+
+	query := client.Query(q)
+	iter, err := query.Read(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	var rows []bigquery.ValueList
+
+	for iter.Next(ctx) {
+		var row bigquery.ValueList
+		if err := iter.Get(&row); err != nil {
+			return nil, err
+		}
+		rows = append(rows, row)
+	}
+
+	return rows, iter.Err()
+}
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("usage: ", os.Args[0], "'query text'")
+		os.Exit(1)
+	}
+	proj := os.Getenv("GCLOUD_PROJECT")
```

Comment:
What do you think of `BIGQUERY_PROJECT_ID`?
I'm really not sure what we should do about this, generally.
I'm also fine with `GCLOUD_PROJECT`.